Makefile

```diff
@@ -55,6 +56,29 @@ else
 	rm temp-test.in
 endif
 	make install-flexmeasures
+# Locally install HiGS on macOS
+	if [ "$(shell uname)" = "Darwin" ]; then \
+		make install-highs; \
+	fi
+
+$(HIGHS_DIR):
+	if [ ! -d $(HIGHS_DIR) ]; then \
+		git clone https://github.com/ERGO-Code/HiGHS.git $(HIGHS_DIR); \
+	fi
+	brew install cmake;
```

Comment:
This line makes all this macOS specific. So we should make this line work both ways, or maybe rename the comnands to make clear this is only meant for macOS

Reply:
This is called for macOS only so I think renaming would be enough